Cargo.toml

```
@@ -29,6 +29,7 @@ members = [
 anyhow = "1"
 const-format = { package = "const_format", version = "0.2" }
 convert-case = { package = "convert_case", version = "0.6" }
+fluent-asserter = "0.1.9"

```

Require "0.1" should be enough
```suggestion
fluent-asserter = "0.1"

```